tools/test-case-generators/format-request-map.json

```diff
@@ -42,6 +68,23 @@
       "image-type": "openstack",
       "repositories": [],
       "filename": "disk.qcow2",
+      "blueprint": {
```

Comment:
This and the later hunk is unrelated, I assume?

Reply:
yea, this is needs to be rebased :/